src/js/utils/helper.js

```diff
@@ -0,0 +1,11 @@
+const isEmptyString = (string) => {
+  if (string === null) {
+    return true;
+  }
+  if (string.trim() === '') {
+    return true;
+  }
+  return false;
+};
```

Comment:
```suggestion
return !string 
```
단순히 boolean을 return 하시는 거면 이렇게 한줄로도 표현하실 수 있어요~!